rules/windows/defense_evasion_create_mod_root_certificate.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2021/02/01"
+maturity = "production"
+updated_date = "2021/02/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or modification of a local trusted root certificate in Windows. The install of a malicious root
+certificate would allow an attacker the ability to masquerade malicious files as valid signed components from any entity
+(e.g. Microsoft). It could also allow an attacker to decrypt SSL traffic.
+"""
+false_positives = ["Certain applications may install root certificates for the purpose of inspecting SSL traffic."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Creation or Modification of Root Certificate"
+references = [
+    "https://posts.specterops.io/code-signing-certificate-cloning-attacks-and-defenses-6f98657fc6ec",
+    "https://www.ired.team/offensive-security/persistence/t1130-install-root-certificate",
+]
+risk_score = 21
+rule_id = "203ab79b-239b-4aa5-8e54-fc50623ee8e4"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+registry where event.type in ("creation", "change") and
+  registry.path :
+    (
+      "HKLM\\Software\\Microsoft\\SystemCertificates\\Root\\Certificates\\*\\Blob",
+      "HKLM\\Software\\Microsoft\\SystemCertificates\\AuthRoot\\Certificates\\*\\Blob",
+      "HKLM\\Software\\Policies\\Microsoft\\SystemCertificates\\Root\\Certificates\\*\\Blob",
+      "HKLM\\Software\\Policies\\Microsoft\\SystemCertificates\\AuthRoot\\Certificates\\*\\Blob"
+    ) and
+  /* add potential noisy FP's here */
+  not process.name : ("svchost.exe", "ssm-agent-worker.exe", "iexplore.exe", "chrome.exe")
```

Comment:
Is there a way to more finely tune these out, since they could just as easily be used as the vector for this abuse?

Reply:
I thought about this as well. I only saw 10 total events in my test environment (including my purposeful mods), but I assumed larger envs may have more activity around web browsers that could cause some noise.  We don't HAVE to exclude anything if the user wants to analyze each event (shouldn't be too noisy).  A sequence wouldn't really be what I want. We could do `not user.id : ("S-1-5-20", "S-1-5-18")` instead which would eliminate any network and SYSTEM user mods but that doesn't eliminate the web browsers though, but that could've just been noise because I installed chrome and made some IE changes on the server I was using. I wasn't get hits testing in other clusters.

Reply:
Removed the process names since it could lead to false negatives